wordpress-mobile/WordPress-iOS
# Reader: Rename `ReaderPostCardCell` to `OldReaderPostCardCell`

Part of #21645 

## Description

Renames `ReaderPostCardCell` to `OldReaderPostCardCell`, there are no changes to functionality with it.

## Testing

To test:
- Launch Jetpack and login
- Tap on the Reader tab
- 🔎 **Verify** the reader post cell loads correctly and works as it previously did

## Regression Notes
1. Potential unintended areas of impact
Reader post card

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
